prog/encoding.go

```diff
@@ -100,6 +100,15 @@ func (a *PointerArg) serialize(ctx *serializer) {
 	}
 	target := ctx.target
 	ctx.printf("&%v", target.serializeAddr(a))
+	if a.Res != nil {
```

Comment:
This block looks wrong/misplaced. It seems that the existing code should handle glob w/o any changes. Or not?

Reply:
No, it won't handle. With the change, it can show the paths choosed, otherwise just &(0x7f0000000000) without showing ```='/sys/power/resume'```

```
r0 = openat$sysfs(0xffffffffffffff9c, &(0x7f0000000000)='/sys/power/resume', 0x2001, 0x0)
r1 = openat$sysfs(0xffffffffffffff9c, &(0x7f0000000000)='/sys/power/resume', 0x0, 0x0)
read(r1, &(0x7f0000000080)=""/71, 0x47)
write(r0, &(0x7f0000000080)='9', 0x3)

```

Reply:
But that's because of the isDefaultArg implementation. You say that whatever value we have, it's the default value, so serialization does not save it because it's assumed that the default value will be restored on load.